lib/GridItem.jsx

```diff
@@ -379,8 +378,7 @@ export default class GridItem extends React.Component<Props, State> {
     const positionParams = this.getPositionParams();
 
     // This is the max possible width - doesn't go to infinity because of the width of the window
-    const maxWidth = calcGridItemPosition(positionParams, 0, 0, cols - x, 0)
-      .width;
+    const maxWidth = calcGridItemPosition(positionParams, 0, 0, cols, 0).width;
```

Comment:
The removal of `cols - x` in the 4th paramater here may cause problems when resizing normally

Reply:
If I don't change that, we can't resize, from the west, the elements of the last column on the right